.buildkite/nightly.steps.yaml

```
@@ -51,10 +51,10 @@ steps:
       message: "exampleproject-4.26 ${BUILDKITE_MESSAGE}"
       env: 
         BUILD_TYPE: "EXAMPLE"
-        GDK_BRANCH: "master"
-        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK"
+        GDK_BRANCH: "0.13.0-rc" # NOTE: temp fix for new ci release interop issues
+        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK-0.13.0-rc" # NOTE: temp fix for new ci release interop issues
         ENGINE_MAJOR: "4.26"
-        PROJECT_BRANCH: "${BUILDKITE_BRANCH}"
+        PROJECT_BRANCH: "0.13.0-rc" # NOTE: temp fix for new ci release interop issues
```

```suggestion
        PROJECT_BRANCH: "${BUILDKITE_BRANCH}"
```
Similarly in the other PRs as well.